module/solutions/helper.js

```diff
@@ -429,12 +440,35 @@ module.exports = class SolutionsHelper {
             message: constants.apiResponses.SOLUTION_NOT_FOUND,
           });
         }
-        if(checkDate){
-          if(solutionData.hasOwnProperty("endDate")){
-            solutionData.endDate = gen.utils.getStartDate(solutionData.endDate)
+
+        let programData = await programsHelper.programDocuments(
```

Comment:
@ankitpws As discussed let's put validation for the start date as well. The solution start date cannot be before the program start date.